app/src/main/java/org/simple/clinic/patient/OngoingNewPatientEntry.kt

```diff
@@ -225,15 +225,17 @@ data class OngoingNewPatientEntry(
 
   @Parcelize
   data class Address(
-      val colonyOrVillage: String, // TODO (rj) 8-Nov-19: Add `streetAddress` and `zone` fields.
+      val colonyOrVillage: String,
       val district: String,
-      val state: String
+      val state: String,
+      val streetAddress: String?,
```

Comment:
Are these nullable because they are optional fields? If so, might make more sense to explicitly use `Optional<String>` instead?

Reply:
In Kotlin, I don't find it useful to use `Optional`. Also I find null handling syntax is much better than using Optionals. So let's keep it as a nullable field?

Reply:
It might make sense to declare this as `Optional` just to expose the meaning at the UI level that the fields are optional.

But I don't feel strongly about this either way. Just wanted to confirm. We can revisit this in the future if needed.